tests/test_csc.py

```diff
@@ -655,12 +670,20 @@ def callback(data):
                     with self.subTest(axis=axis):
                         self.assertEqual(0.0, getattr(offset_reset, axis))
 
-                # Open the loops, mostly so the output is cleaner
-                logger.debug("Disabling corrections")
+                # Open two loops, mostly so the output is cleaner
+                logger.debug("Disabling hexapod and spectrograph corrections")
                 await harness.aos_remote.cmd_disableCorrection.set_start(
-                    disableAll=True, timeout=STD_TIMEOUT
+                    hexapod=True, timeout=STD_TIMEOUT
                 )
 
+                # Send to disabled state
+                await harness.aos_remote.cmd_disable.start()
+
+                # Verify mirror is lowered when transitioning from ENABLED
+                # to DISABLED since corrections are enabled
+                assert harness.pnematics.cmd_m1SetPressure.callback.called
```

Comment:
Same comment as above (but in reverse, I guess), why asserting that `harness.pnematics.cmd_m1SetPressure` was called verify the mirror was lowered?

Reply:
see above